mentors/javascript/n.json

```
@@ -16,13 +16,20 @@
     "avatar_url": null,
     "bio": "I have written JavaScript as part of various JS frameworks, open source projects, and profesionally.  As a MEAN stack developer, JS is my life.  I would identify JS as the language I am most comfortable with at an advanced level.  I also teach JS lessons in webinars from time to time. Please feel free to ask me any questions!"
   },
-  
   {
   "github_username": "nadaa",
   "name": "Nada Ghanem",
   "link_text": null,
   "link_url": null,
   "avatar_url": null,
   "bio": "A continous learner, love technology and nature."
-}
+},
+  {
+    "github_username": "nastassiadanilova",
+    "name": "Nastassia Danilova",
+    "link_text": "GitHub",
+    "link_url": "https://github.com/nastassiadanilova",
+    "avatar_url": null,
+    "bio": "I fell in love with JS and all around it once. I have a lot of experience in different sized projects, but I'll never stop to learn smth new every day. I'll be happy to help you"
```

```suggestion
    "bio": "I am very passionate about JavaScript.  I have a lot of experience in different sized projects, but I'll never stop to learn something new every day. I'll be happy to help you."
```
I agree, it makes more sense to not use unnecessary abbreviations.  I also tweaked your first sentence.  I was not quite sure what you were trying to say, so I hope I got it right.  It is not my intention to put words in your mouth.  Feel free to use as much or as little of my suggestion as you like.